examples/mxnet/sampling/dis_sampling/gcn_trainer.py

```diff
@@ -191,11 +191,12 @@ def main(args):
 
     # initialize graph
     dur = []
-    total_count = 153
     for epoch in range(args.n_epochs):
-        for subg_count in range(total_count):
-            print(subg_count)
-            nf = receiver.recv(g)
+        print('epoch: %d' % epoch)
+        idx = 0
+        for nf in sampler:
```

Comment:
my main concern here is that an epoch might have a different number of mini-bathes when a distributed sampler is used. this might be confusing to people.

Reply:
one possible solution is to maintain a receiving queue for each epoch.

Reply:
@zheng-da The main problem is that we don't know the how many subgraph will be generated in each epoch. This issue comes from the design of sampler API. If we can fix the number of subgraph in the sampler API, this problem can be solved easily. Otherwise, I don't think the receiving queue can address this problem very well, and it will also bring some other system overhead.